tests/test_UtilsTest.py

```diff
@@ -30,3 +33,18 @@ def test_clean_text(self):
     def test_clean_text_invalid(self):
         with self.assertRaises(ValueError):
             clean_text("foo", ["invalid"])
+
+    def test_roman_numeral_regex(self):
+        """Make sure ROMAN_NUMERAL_REGEX matches all numbers between 1-199
+        except 5, 50, 100."""
+        expected = (
+            list(range(1, 5))
+            + list(range(6, 50))
+            + list(range(51, 100))
+            + list(range(101, 200))
+        )
+        actual = sorted(
+            roman.fromRoman(n.upper())
+            for n in exrex.generate(ROMAN_NUMERAL_REGEX)
```

Comment:
Looks like generate is limited to 20 by default. Do we want to set the limit high enough that it'd generate all the needed regexes?

Reply:
That bit me too, but this test passes because `limit` refers to length of wildcards, not number of examples generated. `exrex.generate('[abc]+', limit=5)` will exhaustively generate all strings of `[abc]` up to length 5.